cmd/aries-agent-mobile/go.mod

```diff
@@ -14,4 +14,8 @@ require (
 	nhooyr.io/websocket v1.8.3
 )
 
-replace github.com/hyperledger/aries-framework-go => ../../
+replace (
+	github.com/hyperledger/aries-framework-go => ../../
+	github.com/kilic/bls12-381 => github.com/trustbloc/bls12-381 v0.0.0-20201008080608-ba2e87ef05ef
+	github.com/phoreproject/bls => github.com/trustbloc/bls v0.0.0-20201008085849-81064514c3cc
```

Comment:
How come you're `replacing` these instead of just directly using those specific commits? Are there multiple versions of this dependency being used in different areas?

Reply:
I'm not sure if this answers the question, but we are needing to replace modules due to forking those projects. There is an open issue to remove dependencies on forks.

Reply:
Is there a reason why we're using `replace` instead of just directly importing from `github.com/trustbloc/bls12-381`?

Reply:
No, it's like an indication that this replacement is temporary and should be dropped in the future.